### PR TITLE
Updating readme

### PR DESCRIPTION
**Pull request by kenazk**
@markfuller @jdwelch Can you check the Getting Started steps?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,15 +33,19 @@ aws:
   instance:
     count: 5
 ```
-5. Build the plugin
+3. Build the plugin
 ```
 $ mkdir plugins
```
Comment (jdwelch) [inline README.md]: Looks like this is rendered moot by the addition of the file `plugins/attach.pp`?

File: data.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+aws:
+  region: eu-west-1
+  keyname: aws-key-name
```
Comment (jdwelch) [inline data.yaml]: This should probably look something like an SSH key name to give the clue that's what we're talking about.

Comment (jdwelch) [reply data.yaml]: e.g.`my-instance-ssh.pem`

File: README.md [hunk 3]
```diff
@@ -40,15 +33,19 @@ aws:
...
+$ make
+```
+4. Get the sample manifest
+```
 $ git clone https://github.com/lyraproj/puppet-workflow.git
 $ cp lyraproj/puppet-workflow/puppet/testdata/attach.pp lyraproj/lyra/plugins/
```
Comment (jdwelch) [inline README.md]: Looks like these two steps are rendered moot by the addition of the file `plugins/attach.pp`?

File: README.md [hunk 4]
```diff
@@ -40,15 +33,19 @@ aws:
   instance:
     count: 5
 ```
-5. Build the plugin
+3. Build the plugin
 ```
 $ mkdir plugins
+$ make
```
[the remark this answers is not in the document]
Comment (markfuller) [reply README.md]: The licenses tool we use (https://github.com/pmezard/licenses) does not support go mod.

File: README.md [hunk 5]
```diff
@@ -24,9 +18,8 @@ For a more detailed view of how we think about Lyra, check out our [documentatio
 1. Clone the git repo from your $GOPATH:
```
[the remark this answers is not in the document]
Comment (markfuller) [reply README.md]: The licenses tool we use (https://github.com/pmezard/licenses) does not support go mod.

File: README.md [hunk 6]
```diff
@@ -1,21 +1,15 @@
...
 
-A Workflow can be packaged as a deployable unit and consumed from multiple user interfaces, including a Kubernetes custom resource definition/custom controller, a Pipelines interface, and a CLI interface. 
-
-By separating the authoring of the Workflows from their consumption, Lyra enables Application Service teams to provide tested configurations, embedded governance policies, security validated policies in a single, deployable artifact that Application Developers can provision as dependencies within their containerized application. 
+A Workflow can be packaged as a deployable unit and consumed from any of these interfaces. By separating the authoring of the Workflows from their consumption, Lyra enables Application Service teams to provide tested configurations, embedded governance policies, security validated policies in a single, deployable artifact that Application Developers can provision as dependencies within their containerized application. 
 
 For a more detailed view of how we think about Lyra, check out our [documentation](https://docs.google.com/document/d/1oJwg4LlolC3qlt0xG__xjrz16aYwEyOk8GqyNt5_Gdo/edit?usp=sharing)!
```
Comment (jdwelch) [inline README.md]: Why a Google Doc and not in this repo?

File: README.md [hunk 7]
```diff
@@ -24,9 +18,8 @@ For a more detailed view of how we think about Lyra, check out our [documentatio
 1. Clone the git repo from your $GOPATH: 
 `$ git clone https://github.com/lyraproj/lyra`
 
-3. Add the following lines to data.yaml
+2. Add the following lines to data.yaml
```
Comment (jdwelch) [inline README.md]: Suggest we put these values into the manifest for now; Explaining how `lookup()` works is beyond the scope of "Hello, world!", imo. Also, it doesn't appear that `region`, `keyname`, or `instance.count` get used at all.

File: README.md [hunk 8]
```diff
@@ -73,7 +70,6 @@ Here’s a proposed roadmap for the project. Given the infancy of the project, i
 - [ ] Language X
```
Comment (jdwelch) [inline README.md]: Link to [feature request](https://github.com/lyraproj/lyra/issues/new?template=feature_request.md) Issue template! :)

File: CODE-OF-CONDUCT.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+-   Obvious spammers are banned on first occurrence. If we don’t do this, we’ll have spam all over the place.
+-   People who are committing minor formatting / style infractions will get some education, rather than hammering them in the 3 strikes process.
+-   Contact lyra@puppet.com to report abuse or appeal violations. 
+
+Activity-specific guidelines
+-----
+
+-   [Getting technical help](#getting-technical-help)
+-   [Slack guidelines](#irc-guidelines)
```
Comment (jdwelch) [inline CODE-OF-CONDUCT.md]: Kill this line, since there's no Slack section below

File: CODE-OF-CONDUCT.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+-   Search before you file an issue, and try to avoid filing duplicates by taking a look at whether your issue has already been filed before.
+-   Don't start debates on topics not directly related to the scope of a specific issue. Create another issue.
+-   Remove unnecessary lines when quoting other comments.
+-   Please double check to make sure that the information you are including is public (not confidential), especially in attached log files or screenshots.
+
+### Git / source guidelines
+
+-   If you want to contribute code, start with a discussion on a pull request or issue to make sure that what you want to submit is a good idea and architected in a way that will be useful for others.
+-   Look at existing pull requests and issues to make sure that you aren’t duplicating effort.
+-   Review any existing CONTRIBUTOR.MD files associated with the project.
```
Comment (jdwelch) [inline CODE-OF-CONDUCT.md]: s/CONTRIBUTOR/CONTRIBUTING/; maybe link to it as well?

```suggestion
-   Review any existing [CONTRIBUTING.MD](/CONTRIBUTING.MD) files associated with the project.
```

File: CODE-OF-CONDUCT.md (hunk 9 above)
Comment (kenazk) [reply CODE-OF-CONDUCT.md]: Fixed

File: README.md (hunk 8 above)
Comment (kenazk) [reply README.md]: Added

File: README.md (hunk 7 above)
Comment (kenazk) [reply README.md]: Added

File: README.md (hunk 1 above)
Comment (kenazk) [reply README.md]: Fixed

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (kenazk) [reply README.md]: Found if I restrict the make target to lyra, it works ok.

File: data.yaml (hunk 2 above)
Comment (kenazk) [reply data.yaml]: Just killed the data.yaml instead.

**Pull request merged**